yarn.lock

```diff
@@ -24,10 +24,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/compat-data@npm:^7.17.7, @babel/compat-data@npm:^7.20.1, @babel/compat-data@npm:^7.20.5":
```

Comment:
Is this lockfile deduped? It seems to have a few overlaps. E.g. multiple versions of `esbuild`.

Reply:
It's deduped. Probably other packages using `esbuild` as dependency too?